Updating NR Integration Instructions.

rveitch90 opened the pull request:
https://docs.microsoft.com/en-us/azure/active-directory/active-directory-saas-new-relic-tutorial#configure-azure-ad-single-sign-on - Step 3 implies New Relic will provide a subdomain URL  as a login URL. This is not true. 
Updating to reflect the accurate login URL: https://rpm.newrelic.com/accounts/{acc_id}/sso/saml/login

azurecla commented:
Hi __@aashishr__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (aashishr). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
Changes should go through private repo for validation and staging, plus there is a merge conflict.

rveitch90 committed:
Updating NR Integration Instructions. 

https://docs.microsoft.com/en-us/azure/active-directory/active-directory-saas-new-relic-tutorial#configure-azure-ad-single-sign-on - Step 3 implies New Relic will provide a subdomain URL for as a login URL. This is not true. 
Updating to reflect the accurate login URL.
